code/FulpstationCode/sec_expanded_loadouts/detective_vendor.dm

```diff
@@ -0,0 +1,80 @@
+	icon = 'icons/Fulpicons/Surreal_stuff/disco_elysium.dmi'
+	icon_state = "det"
+	icon_deny = "det-deny"
+	req_access = list(ACCESS_SEC_DOORS, ACCESS_FORENSICS_LOCKERS)
+	products = list(/obj/item/clothing/suit/det_suit/disco = 4,
+					/obj/item/clothing/suit/det_suit/disco/aerostatic = 4,
+					/obj/item/clothing/under/rank/security/detective/disco = 4,
+					/obj/item/clothing/under/rank/security/detective/disco/aerostatic = 4,
+					/obj/item/clothing/neck/tie/detective/disco_necktie = 4,
+					/obj/item/clothing/gloves/color/black/aerostatic_gloves = 4,
+					/obj/item/clothing/shoes/sneakers/disco = 4,
+					/obj/item/clothing/shoes/jackboots/aerostatic = 4,
+					/obj/item/clothing/glasses/sunglasses/disco = 4,
+					/obj/item/clothing/under/rank/security/detective = 4,
+					/obj/item/clothing/under/rank/security/detective/skirt = 4,
+					/obj/item/clothing/suit/det_suit = 4,
+					/obj/item/clothing/head/fedora/det_hat = 4,
+					/obj/item/clothing/gloves/color/black = 4,
+					/obj/item/clothing/under/rank/security/detective/grey = 4,
+					/obj/item/clothing/under/rank/security/detective/grey/skirt = 4,
+					/obj/item/clothing/accessory/waistcoat = 4,
+					/obj/item/clothing/suit/det_suit/grey = 4,
+					/obj/item/clothing/suit/det_suit/noir = 4,
+					/obj/item/clothing/head/fedora = 4,
+					/obj/item/clothing/shoes/laceup = 4,
+					/obj/item/assembly/flash/handheld = 4,
+					/obj/item/flashlight/seclite = 4,
+					/obj/item/detective_scanner = 1,
+					/obj/item/radio/headset/headset_sec = 1,
+					/obj/item/holosign_creator/security = 1,
+					/obj/item/reagent_containers/spray/pepper = 1,
+					/obj/item/clothing/suit/armor/vest/det_suit = 1,
+					/obj/item/storage/belt/holster/detective/full = 1,
+					/obj/item/pinpointer/crew = 1,
+					/obj/item/storage/box/evidence = 1,
+					/obj/item/storage/box/evidence = 12,
+					/obj/item/storage/box/matches = 12,
+					/obj/item/storage/fancy/cigarettes/cigars = 12,
+					/obj/item/reagent_containers/food/drinks/coffee = 12,
+					/obj/item/reagent_containers/food/snacks/donut = 12)
+	contraband = list(/obj/item/clothing/glasses/sunglasses = 2,
+					  /obj/item/storage/fancy/donut_box = 2)
+	premium = list(/obj/item/storage/belt/security/webbing = 5,
+					/obj/item/coin/antagtoken = 1,
+					/obj/item/storage/fancy/cigarettes/cigpack_robustgold = 4,
+					/obj/item/storage/box/gum/nicotine = 2,
+					/obj/item/lighter = 4,
+					/obj/item/clothing/mask/cigarette/pipe = 4,
+					/obj/item/storage/fancy/cigarettes/cigars/havana = 12,
+					/obj/item/storage/fancy/cigarettes/cigars/cohiba = 12)
+
+	refill_canister = /obj/item/vending_refill/detective
+	default_price = 650

```

Comment:
damn, son, ain't this a bit too expensive?

Reply:
That's copy pasta; I don't mind changing it, not a big deal.